cfgov/scripts/http_smoke_test.py

```diff
@@ -108,39 +117,61 @@
     '/policy-compliance/guidance/implementation-guidance/hmda-implementation/',
     '/policy-compliance/guidance/implementation-guidance/mortserv/',
     '/policy-compliance/guidance/implementation-guidance/tila-respa-disclosure-rule/',  # noqa: E501
-    '/about-us/',
-    '/about-us/the-bureau/',
     '/about-us/budget-strategy/',
     '/about-us/payments-harmed-consumers/',
     '/about-us/blog/',
     '/about-us/newsroom/',
     '/about-us/events/',
-    '/activity-log/',
     '/about-us/careers/',
     '/about-us/careers/current-openings/',
     '/about-us/doing-business-with-us/',
     '/about-us/innovation/',
-    '/about-us/contact-us/',
-    '/eregulations/',
-    '/eregulations/1026',
+    '/activity-log/',
+    '/your-story/',
 ]
 
+# call `set` on the combined list to weed out dupes
+FULL_RUN = sorted(set(TOP + APPS))
```

Comment:
Could we change this to something like `FALLBACK_URLS`? I think `FULL_RUN` is a misleading name for this list, since it will only be used in cases where something has gone wrong.

Reply:
Good point.

Reply:
👍